File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        vector<char> ch;
        for(int i = 0 ; i < s.size();i++){
            if(s[i]=='\0'){
                continue;
            }
            if(isalnum(s[i])){
                s[i] = tolower(s[i]);
                ch.push_back(s[i]);
            }   
        }
        int start = 0;
        int end = ch.size()-1;
        while(start<end){
            if(ch[start++]!=ch[end--]){
                return false;
            }
        }
        return true;
        
        
    }
};

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        line=""
        for str in s:
            if (str>='a' and str<='z') or (str>='A' and str<='Z') or (str>='0' and str<='9'):
                line=line+lower(str)
        start = 0
        end = len(line)-1
        while(start<end):
            if line[start]!=line[end]:
                return False
            start+=1
            end-=1
        return True


        